sw/device/tests/dif/meson.build

```diff
@@ -114,4 +114,19 @@ dif_plic_sanitytest_lib = declare_dependency(
   ),
 )
 
-sw_tests += { 'dif_plic_sanitytest': dif_plic_sanitytest_lib }
+dif_uart_sanitytest_lib = declare_dependency(
+  link_with: static_library(
+    'dif_uart_sanitytest_lib',
+    sources: ['dif_uart_sanitytest.c'],
+    dependencies: [
+      dif_uart,
+      sw_lib_mmio,
+      sw_lib_runtime_hart,
+    ],
+  ),
+)
+
+sw_tests += {
+  'dif_plic_sanitytest': dif_plic_sanitytest_lib,
+  'dif_uart_sanitytest': dif_uart_sanitytest_lib,
+}
```

Comment:
What I did when I added the rv_timer sanity test is I actually added a new `sw_tests` line, so that the Meson code looked like this:
```
dif_plic_test = ...
sw_tests += { 'dif_plic': dif_plic_test }

dif_timer_test = ...
sw_tests += { 'dif_timer': dif_timer_test }
```

You don't *have* to do this, but it feels like it makes it harder to forget to add the test to the test table.

Reply:
Sounds good, I will do that.

Reply:
I will do it as a minor follow-up to avoid confusion.